src/satosa/micro_services/processors/scope_remover_processor.py

```diff
@@ -0,0 +1,15 @@
+from ..attribute_processor import AttributeProcessorError
+from .base_processor import BaseProcessor
+
+class ScopeRemoverProcessor(BaseProcessor):
+    """
+    Removes the scope from a given attribute 
+    
+    """
+    def process(self, internal_data, attribute, **kwargs):
+        attributes = internal_data.attributes
+        new_values = []
+        for value in attributes.get(attribute, [None]):
```

Comment:
If `attribute` is not found, `[None]` will be returned and `value` will be set to `None`. Then `value.split()` will fail. We should probably check if the attribute is there, and then process the values. If attribute is not there, I would _raise_ an `AttributeProcessorWarning`.